packages/trader/src/sass/app/_common/components/purchase-button.scss

```diff
@@ -152,6 +178,13 @@
                 background: var(--purchase-main-1);
             }
         }
+
+        @include mobile {
+            @include mobile {
```

Comment:
why do we need to add another `@include mobile` here?

Reply:
Whoops! nice find.